web-client/src/presenter/actions/CaseDetail/forwardMessageAction.ts

```diff
@@ -15,17 +8,18 @@ export const forwardMessageAction = async ({
 
   const docketNumber = get(state.caseDetail.docketNumber);
 
-  const { parentMessageId } = await applicationContext
+  const { attachments, parentMessageId } = await applicationContext
     .getUseCases()
     .forwardMessageInteractor(applicationContext, {
       docketNumber,
       ...form,
+      attachments: [...form.attachments, ...form.draftAttachments],
     });
 
   let messageViewerDocumentToDisplay;
-  if (form.attachments.length) {
+  if (attachments) {
     messageViewerDocumentToDisplay = {
-      documentId: form.attachments[0].documentId,
+      documentId: attachments[attachments.length - 1].documentId,
```

Comment:
should this go back to the first item in the array? why are we intentionally going to the last one in the array here?